### DISCUSSION TITLE
npm not installed with nvm install <version>

### DISCUSSION THREAD

**Discussion by renierl** [coreybutler/nvm-windows#802] [General]
Hi there,

I recently got a new PC for work. I installed nvm off the bat, I then installed the node versions I needed, but npm doesn't seem to get installed with the nvm install <version> command. Shouldn't the installation of a node version include an installation of npm, or am just being a noob here?

![image](https://user-images.githubusercontent.com/78844555/167836772-63b720db-cf05-4efe-ba9a-3370d4b90974.png)

**Comment by coreybutler** [comment]
If anyone is still facing this issue, please open an issue. There were some problems with downloads for some older versions of npm (see https://github.com/npm/cli/issues/4890), which should be resolved now.
